packages/tooltip/src/styledComponents/Tooltip.js

```diff
@@ -1,7 +1,10 @@
 // @flow
 import styled from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
 
-const Container = styled.div`position: relative;`;
```

Comment:
same as above, non prettiefied code shouldnt be commited, a proper installed pre-commit should fix this

Reply:
Indeed pre-commit fixed this.